openqabot/loader/gitea.py

```diff
@@ -418,12 +418,17 @@ def make_incident_from_pr(
         if len(incident["channels"]) == 0:
             log.info("Skipping PR %s, no channels found/considered", number)
             return None
-        if (
-            only_successful_builds
-            and len(incident["failed_or_unpublished_packages"]) > 0
-        ):
-            log.info("Skipping PR %s, not all packages succeeded and published", number)
-            return None
+        if only_successful_builds:
+            if len(incident["failed_or_unpublished_packages"]) > 0:
+                log.info(
+                    "Skipping PR %s, not all packages succeeded and published", number
+                )
```

Comment:
is this the tidied version of it?

Reply:
Me neither. Changing the formatting rules is a bit out of scope for this PR, though.

Reply:
Yes, I just wanted to discuss and understand. I didnt request changes!